src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -832,8 +832,8 @@ protected function send_debug_email() {
 This debugging email is sent when you are using a development version of ClassicPress.
 
 If you think these failures might be due to a bug in ClassicPress, could you report it?
- * Open a thread in the support forums: https://wordpress.org/support/forum/alphabeta
- * Or, if you're comfortable writing a bug report: https://core.trac.wordpress.org/
+ * Open a thread in the support forums: https://docs.classicpress.net/faq-support/
```

Comment:
Shouldn't the text also be adjusted? On that page it is not possible to open anything.

Reply:
Updated according to https://github.com/ClassicPress/ClassicPress/issues/198#issuecomment-435225554 . Happy to change it, if needed.